src/components/Form/ActionButton/Prompt.css

```diff
@@ -30,3 +37,8 @@ button.light.toggle-comment-button {
   max-height: 500px;
   transition: max-height 0.25s ease-in;
 }
+
+.panel .text-area-wrapper {
+  padding: 0.5em 0.875em;
```

Comment:
I quite liked the fact that the distance between the first letter and the corner is the same, before: 
![image](https://user-images.githubusercontent.com/4190728/97893487-a4c66300-1d31-11eb-9078-6e0aa5c44af6.png)
and after:
![image](https://user-images.githubusercontent.com/4190728/97893653-db9c7900-1d31-11eb-9b77-1705d73d264d.png)

this avoid a glitchy jump effect when toogling full edit mode.

Why did you add this wrapper btw ?